pyfolio/tears.py

```diff
@@ -1217,3 +1229,161 @@ def create_bayesian_tear_sheet(returns, benchmark_rets=None,
     plt.show()
     if return_fig:
         return fig
+
+
+@plotting_context
+def create_risk_tear_sheet(positions,
+                           style_factor_panel,
+                           sectors=None,
+                           caps=None,
+                           shares_held=None,
+                           volumes=None,
+                           percentile=None,
+                           return_fig=False):
+    '''
+    Creates risk tear sheet: computes and plots style factor exposures, sector
+    exposures, market cap exposures and volume exposures.
+
+    Parameters
+    ----------
+    positions : pd.DataFrame
+        Daily equity positions of algorithm, in dollars.
+        - DataFrame with dates as index, equities as columns
+        - Last column is cash held
+        - Example:
+                     Equity(24   Equity(62
+                       [AAPL])      [ABT])             cash
+        2017-04-03	-108062.40 	  4401.540     2.247757e+07
+        2017-04-04	-108852.00	  4373.820     2.540999e+07
+        2017-04-05	-119968.66	  4336.200     2.839812e+07
+
+    style_factor_panel : pd.Panel
+        Panel where each item is a DataFrame that tabulates style factor per
+        equity per day.
+        - Each item has dates as index, equities as columns
+        - Example item:
+                     Equity(24   Equity(62
+                       [AAPL])      [ABT])
+        2017-04-03	  -0.51284     1.39173
+        2017-04-04	  -0.73381     0.98149
+        2017-04-05	  -0.90132	   1.13981
+
+    sector : pd.DataFrame
+        Daily Morningstar sector code per asset
```

Comment:
What if someone passes non-morningstar data? Maybe there's a way where they can pass strings instead of numbers, then we can plot the strings if so? Could also make that the default, then convert numbers to strings on our end before calling the tear sheet.